pkg/storaged/stratis-panel.jsx

```diff
@@ -61,16 +61,16 @@ function stratis_pool_row(client, path) {
     };
 }
 
-function stratis_locked_pool_row(client, uuid) {
-    const action = <StorageButton onClick={() => unlock_pool(client, uuid, true)}><UnlockIcon /></StorageButton>;
+function stratis_stopped_pool_row(client, uuid) {
+    const action = <StorageButton onClick={() => start_pool(client, uuid, true)}><PlayIcon /></StorageButton>;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230420-131050-f1115ac6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)